### PR TITLE
services/filestore: migrate s3store_test to testify

### PR DESCRIPTION
**Pull request by isacikgoz**
#### Summary

change `t.Fatal` method to `require` functions in `services/filesstore/s3store_test.go`

#### Ticket Link

Fixes: #12664

### CODE REVIEW

File: services/filesstore/s3store_test.go [hunk 1]
```diff
@@ -7,26 +7,24 @@ import (
...
 )
 
 func TestCheckMandatoryS3Fields(t *testing.T) {
 	cfg := model.FileSettings{}
 
 	err := CheckMandatoryS3Fields(&cfg)
-	if err == nil || err.Message != "api.admin.test_s3.missing_s3_bucket" {
-		t.Fatal("should've failed with missing s3 bucket")
-	}
+	res := err == nil || err.Message != "api.admin.test_s3.missing_s3_bucket"
```
Comment (jespino) [inline services/filesstore/s3store_test.go]: You can replace it using `require.NotNil(t, err)` and `require.Equal(t, err.Message, "api.admin.test_s3.missing_s3_bucket")`

File: services/filesstore/s3store_test.go [hunk 2]
```diff
@@ -7,26 +7,24 @@ import (
...
+	require.False(t, res, "should've failed with missing s3 bucket")
 
 	cfg.AmazonS3Bucket = model.NewString("test-mm")
 	err = CheckMandatoryS3Fields(&cfg)
-	if err != nil {
-		t.Fatal("should've not failed")
-	}
+	require.Nil(t, err, "should've not failed")
```
Comment (jespino) [inline services/filesstore/s3store_test.go]: This error string doesn't give any extra information, you can remove it.

File: services/filesstore/s3store_test.go [hunk 3]
```diff
@@ -7,26 +7,24 @@ import (
...
 
 	cfg.AmazonS3Endpoint = model.NewString("")
 	err = CheckMandatoryS3Fields(&cfg)
-	if err != nil || *cfg.AmazonS3Endpoint != "s3.amazonaws.com" {
-		t.Fatal("should've not failed because it should set the endpoint to the default")
-	}
+
+	res = err != nil || *cfg.AmazonS3Endpoint != "s3.amazonaws.com"
```
Comment (jespino) [inline services/filesstore/s3store_test.go]: Same as before, you can use `require.Nil(t, err)` and a `require.Equal(t, *cfg.AmazonS3Endpoint, "s3.amazonaws.com")`

File: services/filesstore/s3store_test.go (hunk 1 above)
Comment (isacikgoz) [reply services/filesstore/s3store_test.go]: I changed this one according to your suggestion.

File: services/filesstore/s3store_test.go (hunk 2 above)
Comment (isacikgoz) [reply services/filesstore/s3store_test.go]: Fail message removed as you suggested.

File: services/filesstore/s3store_test.go (hunk 3 above)
Comment (isacikgoz) [reply services/filesstore/s3store_test.go]: Applied 👍

**Comment by isacikgoz** [comment]
Thank you for reviewing my PR @jespino! I implemented your suggestions 👍

**Comment by jespino** [comment]
@isacikgoz please, merge the master branch here.

**Comment by isacikgoz** [comment]
Done @jespino 👍

**Pull request merged**